### ISSUE TITLE
R101 config

### ISSUE THREAD

**Issue by NaomiEX** [exiawsh/StreamPETR#173]
Hi, I was wondering whether there are any plans to release the R101 configs? or otherwise what changes to the config you would suggest for recreating the results?

**Comment by exiawsh** [comment]
You can use the pretrained weights here:
https://download.openmmlab.com/mmdetection3d/v0.1.0_models/nuimages_semseg/cascade_mask_rcnn_r101_fpn_1x_nuim/cascade_mask_rcnn_r101_fpn_1x_nuim_20201024_134804-45215b1e.pth
You can refer to SimMOD to write the config.
https://github.com/zhangyp15/SimMOD/blob/239d20d82242c4e5253f47e94d525ac2bce1696e/projects/configs/simmod/simmod_r101_nuimg.py#L49

**Comment by NaomiEX** [comment]
thank you for the response!

One question is what about the learning rate? if, say, I have a batch size of 8 or 16 do I use the normal learning rate as the R50 configs or do I use a lower learning rate?

**Comment by exiawsh** [comment]
Keep the same with r50 config.
total bs 16 lr 4e-4
total bs 8 lr 2e-4
